src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/MirrorFetchException.java

```diff
@@ -0,0 +1,8 @@
+package com.atlassian.bitbucket.jenkins.internal.scm;
+
+public class MirrorFetchException extends RuntimeException {
```

Comment:
As I've mentioned elsewhere, I think this should extend the `BitbucketClientException`

Reply:
Fixed